app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -239,6 +243,27 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
                         .zoom(ZOOM_LEVEL)
                         .build();
                 mapBoxMap.setCameraPosition(cameraPosition);
+                scaleBarPlugin = new ScaleBarPlugin(mapView, mapBoxMap);
+
+                int color = isDarkTheme?R.color.bottom_bar_light:R.color.bottom_bar_dark;
+
+                final float SIZE = new TextView(getContext()).getTextSize()-1;     //Default text size for scalebar params
+                final float SCALEBAR_HEIGHT = (float) (SIZE/3.5);                  //these values are chosen to keep scalebar size comparable to other text depending upon the screen size and device resolution
+                final float SCALEBAR_BORDER_WIDTH = SIZE/10;
+                final float SCALEBAR_MARGIN = SIZE/2;
+
+                scaleBarOptions = new ScaleBarOptions(getContext())
+                    .setTextColor(color)
+                    .setTextSize(SIZE)
+                    .setBarHeight(SCALEBAR_HEIGHT)
+                    .setBorderWidth(SCALEBAR_BORDER_WIDTH)
+                    .setRefreshInterval(15)            //Refresh time for scalebar in millisec
+                    .setMarginTop(SCALEBAR_MARGIN)
```

Comment:
Is there an issue with removing this call altogether? I mean this:

```suggestion
                    .setBorderWidth(SCALEBAR_BORDER_WIDTH)
                    .setMarginTop(SCALEBAR_MARGIN)
```

Reply:
No, i can remove and it will assume default values. Even margin width and height can be set to default but that looked quite small in bigger screen devices so i thought of setting values.
Here is image of how it looks Let me know if you want all values to be set to default.

![image](https://user-images.githubusercontent.com/45694892/76728996-1f7c4700-677e-11ea-98c5-72da32c17dc7.png)

Reply:
@318anushka what sivraam is talking abut here is only the line `.setRefreshInterval(15)`

If the default value for that line is actually the same then there is no point us setting it.

>Even margin width and height can be set to default but that looked quite small in bigger screen devices so i thought of setting values.

This was a good instinct these defaults needed to change for our application so keep these changes. Or at least I don't like the look of that default bar

Reply:
> @318anushka what sivraam is talking abut here is only the line `.setRefreshInterval(15)`
> 
> If the default value for that line is actually the same then there is no point us setting it.
> yes i get it, just thought should let you know how all default settings look like.
i'll make necessary changes 👍 
> > Even margin width and height can be set to default but that looked quite small in bigger screen devices so i thought of setting values.
> 
> This was a good instinct these defaults needed to change for our application so keep these changes. Or at least I don't like the look of that default bar

Reply:
> @318anushka what sivraam is talking abut here is only the line `.setRefreshInterval(15)`
> 
> If the default value for that line is actually the same then there is no point us setting it.

yes i get it , just to let you know how all default settings look like. I'll make necessary changes 👍